src/serialization.cpp

```diff
@@ -160,6 +169,24 @@ static void serializeContent(std::string* content, ov::Tensor& tensor) {
     }
 }
 
+static void serializeStringContent(std::string* content, ov::Tensor& tensor) {
```

Comment:
```suggestion
static void serializeStringContent(std::string* content, ov::Tensor& tensor) {
    OVMS_PROFILE_FUNCTION();
    // We only fill if the content is not already filled.
    // It can be filled in gather exit node handler.
    if (!content->empty()) {
        return;
    }

    size_t batchSize = tensor.get_shape()[0];
    size_t maxStringLen = tensor.get_shape()[1];
    for (size_t i = 0; i < batchSize; i++) {
        // std::string tmp((char*)tensor.data() + i * tensor.get_shape()[1]);
        uint32_t strLen = strnlen((char*)tensor.data() + i * maxStringLen, maxStringLen);
        content->append(reinterpret_cast<const char*>(&strLen), sizeof(strLen));
        content->append((char*)tensor.data() + i * maxStringLen, strLen);
    }
}
```

This way we can avoid unneccessary string copy